src/demo/SidebarSections.tsx

```diff
@@ -134,8 +128,37 @@ export const SdkOptionsView: FunctionComponent<{
           />
         ))}
       </div>
-    </div>
-
+    </label>
+    <label>
+      crossDeviceClientIntroProductName
+      <div>
+        <input
+          type="text"
+          value={sdkOptions.crossDeviceClientIntroProductName || ''}
```

Comment:
What if the string passed is `"/><script>alert('xss')</script>""`?

Reply:
It will get displayed as a plain string and the alert doesn't get executed since these are in JSX.